src/library_pthread.js

```diff
@@ -441,7 +441,7 @@ var LibraryPThread = {
     allocateUnusedWorker() {
       var worker;
 #if MINIMAL_RUNTIME
-      var pthreadMainJs = Module['worker'];
+      var pthreadMainJs = Module['worker'] || './{{{ PTHREAD_WORKER_FILE }}}';
```

Comment:
Do we need both here?

Reply:
I'm honestly not sure.. its possible its being used I suppose?

All this code is hopefully going away soon with #21701 anyway..

Reply:
Did you add this to fix the test? I'm just trying to understand why this change appears here.

Reply:
I added this to make the tests work yes.  Without this we were depending on the HTML to set this.. which doesn't exist when running a test under node.

Reply:
(See the code I removed from the HTML file)

Reply:
I see, thanks, I didn't connect this to that other place. Makes sense now.